wagtail/coreutils.py

```diff
@@ -327,17 +328,21 @@ def get_supported_content_language_variant(lang_code, strict=False):
     raise LookupError(lang_code)
 
 
-@functools.lru_cache
```

Comment:
Whats the reasoning behind the switch to cache backend here? I feel like `lru_cache` is appropriate mechanism?

Reply:
Because you can't reliably clear an `lru_cache`, where you can with a Django cache. Whether this needs a cache at all

Reply:
Scrolls down and sees additions to `signal_handlers.py` changes. Feels a little silly 😀